react-sharing-state/my-app/src/Container.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useState } from 'react';
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ */
+export default function Container({ items }) {
+  const [index, setIndex] = useState(0);
+
+  return (
+    <div>
+      <div>{items[index]}</div>
+      <div>
+        <CustomButton
+          text="Prev"
+          handleClick={() => {setIndex((index - 1 + items.length) % items.length)}}
+           />
+        <Indicators
+          count={items.length}
+          index={index}
+          onSelect={(i) => setIndex(i)}
+          />
+        <CustomButton
+          text="Next"
+          handleClick={() => {setIndex((index + 1) % items.length)}}
+         />
+      </div>
+    </div>
+  );
+}
+
+/**
+ * A component that wraps a DOM button.
+ * Props:
+ *   text: The button's text
+ */
+function CustomButton({ text, handleClick, backgroundColor}) {
+  return <button
+            style={{backgroundColor}}
```

Comment:
I made this work through trial and error. 
I tried to use another variable, like `color` and then have `style={{backgroundColor: {color}}}`
Why did this not work?